rust/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+pub mod do_username {
+	use rand::Rng;
+	fn to_title_case(s: &str) -> String {
+		let mut c = s.chars();
+		match c.next() {
+			None => String::new(),
+			Some(f) => f.to_uppercase().chain(c).collect(),
+		}
+	}
+	
+	fn random(arr: Vec<&str>) -> &str {
+		let mut rng = rand::thread_rng();
+		let r = rng.gen_range(0, arr.len());
+		arr[r]
+	}
+	fn up_format(arr: Vec<&str>) -> String {
+		let mut new_arr = vec![];
+		for elem in arr {
+			new_arr.push(to_title_case(elem))
+		}
+		new_arr.join("")
+	}
+
+	pub fn get() -> String {
```

Comment:
```suggestion
	pub fn generate(size: Option<usize>) -> String {
	  let max_size = size.unwrap_or(30);
```
Method should be named `generate`, per main README, and should support an optional size param.